backend/fms_core/migrations/0070_v5_1_0.py

```diff
@@ -0,0 +1,138 @@
+
+from django.db import migrations
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+UNKNOWN_INDEX_NAME = "UNKNOWN_INDEX"
+def create_empty_index(apps, schema_editor):
+    Index = apps.get_model("fms_core", "Index")
+    IndexStructure = apps.get_model("fms_core", "IndexStructure")
+    Sequence = apps.get_model("fms_core", "Sequence")
+
+    NO_FLANKERS_INDEX_STRUCTURE = {
+        "name": "No_Flankers_Index_Structure",
+        "flanker_5prime_forward": "",
+        "flanker_5prime_reverse": "",
+        "flanker_3prime_forward": "",
+        "flanker_3prime_reverse": ""
+    }
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        reversion.set_comment(f"Create index '{UNKNOWN_INDEX_NAME}' and its structure '{NO_FLANKERS_INDEX_STRUCTURE['name']}'.")
+        reversion.set_user(admin_user)
+
+        # Create structure
+        flanker_5prime_forward, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_5prime_forward"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        flanker_5prime_reverse, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_5prime_reverse"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        flanker_3prime_forward, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_3prime_forward"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        flanker_3prime_reverse, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_3prime_reverse"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        index_structure = IndexStructure.objects.create(
+            name=NO_FLANKERS_INDEX_STRUCTURE["name"],
+            flanker_5prime_forward=flanker_5prime_forward,
+            flanker_5prime_reverse=flanker_5prime_reverse,
+            flanker_3prime_forward=flanker_3prime_forward,
+            flanker_3prime_reverse=flanker_3prime_reverse,
+            created_by_id=admin_user.id,
+            updated_by_id=admin_user.id
+        )
+        reversion.add_to_revision(flanker_5prime_forward)
```

Comment:
You are adding the same pre-existing sequence four times to the revisions ...

Reply:
done.